src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCancelByIdControllerTest.java

```diff
@@ -92,195 +86,8 @@ public void user_should_cancel_task_when_grant_type_specific_and_permission_canc
         common.cleanUpTask(taskId);
     }
 
-    @Test
-    public void user_should_cancel_task_when_grant_type_specific_and_permissions_read_manage_cancel() {
-
-        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json",
-                                                                       "reviewSpecificAccessRequestJudiciary");
-        String taskId = taskVariables.getTaskId();
-
-        common.setupHearingPanelJudgeForSpecificAccess(caseworkerCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-
-        initiateTask(taskVariables, caseworkerCredentials.getHeaders());
-
-        Response result = restApiActions.post(
-            ENDPOINT_BEING_TESTED,
-            taskId,
-            caseworkerCredentials.getHeaders()
-        );
-
-        result.then().assertThat()
-            .statusCode(HttpStatus.NO_CONTENT.value());
-
-        common.cleanUpTask(taskId);
-    }
-
-    @Test
-    public void user_should_cancel_task_when_grant_type_specific_and_permissions_read_manage_own_cancel() {
```

Comment:
where is this now verified?

Reply:
Required permission in CANCEL. This tests doesn't do anything extra as possitive scenarion for same grant type is tested in above test. (Can add one to cover all in controller IT test though)

Reply:
Can add one to cover all in controller CFT IT test in new ticket